### DISCUSSION TITLE
Build without HTTP_DAEMON and kill

### DISCUSSION THREAD

**Discussion by rsbeckerca** [openssl/openssl#22137] [Q&A]
I am trying to port OpenSSL to an environment that does not have the `kill()` call or pids in the same form as POSIX. Is there any way to configure for that?

**Comment by t8m** [comment]
Configuring with no-cmp could help.

**Reply by rsbeckerca** [reply]
Sadly no. I still get HTTP_DAEMON defined, which invokes `kill` in `apps/lib/http_server.c`.

**Reply by rsbeckerca** [reply]
@t8m I'm going to try defining `OPENSSL_NO_SOCK` for the `nonstop-nsx_g` config and see whether that helps.

**Reply by rsbeckerca** [reply]
That didn't work. I'm going to look at other code options.
